.github/workflows/tact.yml

```diff
@@ -231,7 +231,14 @@ jobs:
       - name: (pnpm) Test creation of new Blueprint projects
         if: ${{ matrix.package-manager == 'pnpm' }}
         run: |
-          npm i -g pnpm
+          npm i -g pnpm@9.12.2
```

Comment:
let's add a link or something to explain this change, because it looks pretty involved